### ISSUE TITLE
Custom button in Toolsbar (specific picto and function)

### ISSUE THREAD

**Issue by Xavlight** [fengyuanchen/viewerjs#490]
Hello,

I am looking to add a specific button to the toolbar.
This button could call a specific function.

The idea would be to have a 'delete' button to start the delete procedure in the database.

Thanks

**Comment by fengyuanchen** [comment]
Please follow this example (which implement a custom download button on the toolbar): 

> https://fengyuanchen.github.io/viewerjs/examples/custom-toolbar.html

**Comment by Xavlight** [comment]
Hi fengyuanchen,
Yes, I did see that example about customising the toolbar, but only using the native viewer pictos and functions.

I want to create my own button associated with my own function.

I found a solution by adding a line <li> at the end of the list of buttons <ul>.
```
var viewer = new Viewer(images, {
	ready() {
		$('.viewer-toolbar ul').append('<li role="button" class="far fa-trash-alt viewer-large viewer-delete" onclick="myFunction();return false;"></li>');
	},
	....
```

$('.viewer-toolbar ul').append('<li role="button" class="far fa-trash-alt viewer-large viewer-delete" onclick="myFunction();return false;"></li>');

```
css :
	.viewer-delete {
	background-color:red!important;
	color:white;
	font-size:20px;
	line-height:30px;
}
```
It works perfectly.

Maybe there are other ways to do it?

Thanks
